src/api/routes/articleRouter.js

```diff
@@ -48,4 +50,28 @@ articleRouter.delete('/:slug',
   checkArticleOwner.checkOwner,
   articleController.delete);
 
+articleRouter.get('/:slug/share/facebook',
+  checkValidToken,
+  checkArticle.getArticle,
```

Comment:
what is the user getting?? I think the method is POST

Reply:
yeah you are right